src/Aspire.Hosting.Azure.EventHubs/AzureEventHubsExtensions.cs

```diff
@@ -133,6 +134,15 @@ public static IResourceBuilder<AzureEventHubsResource> RunAsEmulator(this IResou
         }
 
         // Add emulator container
+        var configHostFile = Path.GetTempFileName();
+        File.WriteAllBytes(configHostFile, []);
```

Comment:
Why is this necessary? `Path.GetTempFileName` is documented as

https://learn.microsoft.com/en-us/dotnet/api/system.io.path.gettempfilename

> Creates a uniquely named, zero-byte temporary file on disk and returns the full path of that file.